### PR TITLE
Migrated reminders page to b3

### PR DESCRIPTION
**Pull request by kkrampa**
@biyeun

### CODE REVIEW

**Comment by biyeun** [comment]
@kkrampa more merge conflicts @_@

**Comment by gcapalbo** [comment]
(also @biyeun, I haven't had a chance to test this one locally yet, will test it tomorrow)

**Comment by gcapalbo** [comment]
(didn't get to test this one today but tested the keywords and the supply sms alerts ones, will test this one tomorrow)

File: corehq/apps/reminders/forms.py [hunk 1]
```diff
@@ -801,26 +840,36 @@ def section_repeat(self):
...
                 _("Additional Stop Condition"),
-                InlineField(
-                    'stop_condition',
-                    data_bind="value: stop_condition",
-                    css_class="input-xlarge",
+                crispy.Div(
+                    InlineField(
+                        'stop_condition',
+                        data_bind="value: stop_condition",
+
```
Comment (gcapalbo) [inline corehq/apps/reminders/forms.py]: we can probably get rid of this empty line

**Comment by gcapalbo** [comment]
@kkrampa i tested the reminder list and single event reminder uis and below is the feedback. i haven't yet had a chance to test the multi-event reminder ui but will do so in the coming days

reminder list:
- the activate and deactivate buttons should be side by side, not one on top of the other

single event reminder ui:
- the "Only Cases in Following State" choice is not completely visible in the Send Reminder For select box
- when you select "Date in Case" for Day of Reminder, the "Before Date By" text is not completely visible in the dropdown input
- the "Last User who Modified Case" text is not completely visible in the Recipient dropdown
- when "Specific Child Case" is chosen for the Recipient, the child case filter inputs are extremely small
- there is no label for the "Submit Partial Forms" advanced option
- the label for "Use Custom Content Handler" is not placed correctly and is making the text be srunched
- validation error messages aren't being displayed for many of the fields (you can see which ones by leaving them blank and trying to save, they include the "when case property" and "time of day" fields)
- when "send for case type" is left blank, the validation error message shows up but it causes the "send reminder for" input to be displayed to the right of it instead of underneath it

**Comment by biyeun** [comment]
@kkrampa more conflicts

**Comment by kkrampa** [comment]
@gcapalbo Regarding missing labels, here is screen from old version: 
image

**Comment by kkrampa** [comment]
Updated

**Comment by biyeun** [comment]
@kkrampa can you update conflicts and then I can put this on staging to get QA'd

**Comment by gcapalbo** [comment]
@kkrampa Looks good. I tested the multi-event reminder ui and it looks good too. There's still just one thing outstanding - when I select "Specific Child Case" as the recipient, the inputs that are displayed are very small.

**Comment by kkrampa** [comment]
@gcapalbo @biyeun Updated

**Comment by gcapalbo** [comment]
@biyeun This is good from my side - was there any other feedback you had or can I merge?

**Comment by biyeun** [comment]
looks good to me

**Pull request merged**